internal/console/scan.go

```diff
@@ -22,18 +24,57 @@ import (
 	"github.com/rs/zerolog"
 	"github.com/rs/zerolog/log"
 	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
 )
 
 var scanCmd = &cobra.Command{
 	Use:   "scan",
 	Short: "Executes a scan analysis",
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		return initializeConfig(cmd)
+	},
 	RunE: func(cmd *cobra.Command, args []string) error {
 		return scan()
 	},
 }
 
+func initializeConfig(cmd *cobra.Command) error {
+	v := viper.New()
```

Comment:
I think we should check if `cfgFile` has any value to execute initialize and, if has a value, but the path is wrong should return an exiting error.

Reply:
resolved